### ISSUE TITLE
Choose Template "Tree" Keyboard Tabbing?

### ISSUE THREAD

**Issue by jpdevries** [jpdevries/modx-create-resource#2]
Does anyone know if this should be considered a browser bug? I’m using a `<details>` `<fieldset>` pattern and when a `<summary>` blurs (tabbing), it sets focus to the nested `<summary>` rather than the `input` even though the input is first. In other words, it is skipping over inputs.
https://codepen.io/jpdevries/pen/ybwvZp?editors=1100
